README.md

```diff
@@ -43,6 +46,7 @@ $ http --json ifconfig.co
 {
   "city": "Bornyasherk",
   "country": "Elbonia",
+  "isoCountry": "EB",
```

Comment:
Field should follow same naming convention as other fields. Can you make it `country_iso`?